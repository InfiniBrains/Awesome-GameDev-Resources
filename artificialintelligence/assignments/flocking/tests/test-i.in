3.000 3.000 3.000 1.000 1.000 2.000 3.000 2
1.000 -0.500 -1.500 1.500
-1.000 0.500 -1.000 1.000
0.125
0.125
