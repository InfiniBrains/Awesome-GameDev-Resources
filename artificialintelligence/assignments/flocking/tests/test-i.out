0.746 -0.251 -2.034 1.992
-1.175 0.681 -1.403 1.445
0.401 0.082 -2.760 2.661
-1.421 0.939 -1.967 2.065
